scripts/scil_save_connections_from_hdf5.py

```diff
@@ -39,6 +42,17 @@ def _build_arg_parser():
     p.add_argument('--include_dps', action='store_true',
                    help='Include the data_per_streamline the metadata.')
 
+    group = p.add_mutually_exclusive_group()
+    group.add_argument('--edge_keys', nargs='+',
+                       help='Keys to identify the sub-network of interest.')
```

Comment:
Replace sub-network by edge maybe ?
Also an example like `Keys to identify the edges of interest (LABEL1_LABEL2).`